exporters/zipkin/src/test/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterEndToEndHttpTest.java

```diff
@@ -148,7 +143,10 @@ private void exportAndVerify(ZipkinSpanExporter zipkinSpanExporter) {
         .setStartEpochNanos(START_EPOCH_NANOS)
         .setAttributes(attributes)
         .setTotalAttributeCount(attributes.size())
-        .setEvents(annotations)
+        .setEvents(
+            Arrays.asList(
+                EventData.create(RECEIVED_TIMESTAMP_NANOS, "RECEIVED", Attributes.empty()),
+                EventData.create(SENT_TIMESTAMP_NANOS, "SENT", Attributes.empty())))
```

Comment:
That does not have the same behavior. For example if by mistake the exporter mutates the events list this test and all the test actually will no longer catch that.